src/modules/m_override.cpp

```diff
@@ -84,6 +98,17 @@ class ModuleOverride : public Module
 		ConfigTag* tag = ServerInstance->Config->ConfValue("override");
 		NoisyOverride = tag->getBool("noisy");
 		RequireKey = tag->getBool("requirekey");
+		bool ShouldEnableUmode = tag->getBool("enableumode", false);
+		if (ShouldEnableUmode && !UmodeEnabled)
+		{
+			ServerInstance->Modules->AddService(ou);
```

Comment:
Changing this on rehash is problematic because it may cause desyncs if one server has the mode and another does not. You should enable it in the mode constructor at load time like m_redirect does.